include/secp256k1_extrakeys.h

```diff
@@ -72,13 +70,13 @@ SECP256K1_API int secp256k1_xonly_pubkey_serialize(
  *  Returns: 1 if the public key was successfully converted
  *           0 otherwise
  *
- *  Args:         ctx: pointer to a context object (cannot be NULL)
+ *  Args:         ctx: pointer to a context object.
  *  Out: xonly_pubkey: pointer to an x-only public key object for placing the
- *                     converted public key (cannot be NULL)
- *          pk_parity: pointer to an integer that will be set to 1 if the point
- *                     encoded by xonly_pubkey is the negation of the pubkey and
- *                     set to 0 otherwise. (can be NULL)
- *  In:        pubkey: pointer to a public key that is converted (cannot be NULL)
+ *                     converted public key.
+ *          pk_parity: Optional(if NULL): pointer to an integer that
```

Comment:
How about `Ignored if NULL. Otherwise, pointer ....`

Reply:
That's better, Thanks!